File: scripts/check_matches.py
import os
from supabase import create_client
from dotenv import load_dotenv

load_dotenv()
supabase = create_client(os.getenv('SUPABASE_URL'), os.getenv('SUPABASE_KEY'))

response = supabase.table('matches').select('*').execute()
print(f'Total Matches in DB: {len(response.data)}')
for m in response.data:
    if m['result'] is None:
        print(f"UPCOMING: {m['home_team']} vs {m['away_team']} at {m['kickoff']}")
    else:
        print(f"FINISHED: {m['home_team']} vs {m['away_team']} (Result: {m['result']})")

File: web/check_matches.js
const { createClient } = require('@supabase/supabase-js');
require('dotenv').config({ path: '.env.local' });

const supabase = createClient(process.env.NEXT_PUBLIC_SUPABASE_URL, process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY);

async function main() {
  const { data, error } = await supabase.from('matches').select('*').order('kickoff');
  if (error) console.error(error);
  else console.table(data.map(m => ({ id: m.id, home: m.home_team, away: m.away_team, kickoff: m.kickoff, result: m.result })));
}

main();
